apps/server/src/modules/board/service/drawing-element-adapter.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { LegacyLogger } from '@src/core/logger';
+import { firstValueFrom } from 'rxjs';
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+import { HttpService } from '@nestjs/axios';
+
+@Injectable()
+export class DrawingElementAdapterService {
+	constructor(private logger: LegacyLogger, private readonly httpService: HttpService) {
+		this.logger.setContext(DrawingElementAdapterService.name);
+	}
+
+	async deleteDrawingBinData(drawingName: string): Promise<void> {
+		await firstValueFrom(
```

Comment:
I do agree with your concern here https://github.com/hpi-schul-cloud/schulcloud-server/pull/4352#discussion_r1394183735 that the deletion endpoint on the tldraw side should not be left completely open. Have you checked if there is an administration api that allows you to do all deletions with a single, seperate user?

Reply:
After conv with devops, we just closed possibility of accesing tldraw-document constoller from external addresses, so now if we authorize beforehand, this endpoind can stay as it is.